### ISSUE TITLE
Is there a way to use annotorious without save/cancel popup?

### ISSUE THREAD

**Issue by andrewkouri-old** [annotorious/annotorious-v1#177]
Would like to use Annotorious just for the bounding boxes, however the save and cancel popups are causing repetitive motion injury.

**Comment by forthtemple** [comment]
Yes, I really need this too. The code isnt trivial to get my head around. Was hoping could just look through the code and see where could disable the popup

**Comment by forthtemple** [comment]
In editor.js in the function 

`annotorious.Editor.prototype.open = function(opt_annotation, opt_event) `

under the line

`this._annotator.fireEvent(annotorious.events.EventType.EDITOR_SHOWN, opt_annotation);`

put the line
`this._btnSave.click();`

This will automatically close the annotation editor box with no comment
